### PR TITLE
Added some comments in CMakeLists.txt

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
as title. updated some comments

### CODE REVIEW

**Comment by dutor** [comment]
Suspend this PR until #1332 is merged

**Comment by dutor** [comment]
Now there are conflicts, please resolve them first.

File: CMakeLists.txt [hunk 1]
```diff
@@ -3,24 +3,26 @@
...
+#   NEBULA_OTHER_ROOT              -- Specify the root directory for user building
 #                                  -- Split with ":", exp: DIR:DIR
 #   NEBULA_CLANG_USED_GCC_TOOLCHAIN           -- Set the gcc installation directory used by Clang(libstdc++)
 #
 #   ENABLE_JEMALLOC                -- Link jemalloc into all executables
 #   ENABLE_NATIVE                  -- Build native client
 #   ENABLE_TESTING                 -- Build unit test
 #
-cmake_minimum_required(VERSION 3.5.0)
+cmake_minimum_required(VERSION 3.0.0)  # CMake version check
```
Comment (darionyaphet) [inline CMakeLists.txt]: why downgrade from `3.5.0` to `3.0.0`?

File: CMakeLists.txt [hunk 2]
```diff
@@ -3,24 +3,26 @@
...
 
-project("Nebula Graph" C CXX)
+project("Nebula Graph" C CXX)          # Set the project name
```
Comment (darionyaphet) [inline CMakeLists.txt]: maybe we should put the comment before the source code for example:

```
# Set the project name
project("Nebula Graph" C CXX)
```

**Comment by dutor** [comment]
Jenkins go

**Comment by dutor** [comment]
Jenkins go

**Pull request merged**